src/DynamoCoreWpf/ViewModels/Core/DynamoViewModelDelegateCommands.cs

```diff
@@ -122,6 +122,9 @@ private void InitializeDelegateCommands()
         public DelegateCommand SaveImageCommand { get; set; }
         public DelegateCommand ShowSaveImageDialogAndSaveResultCommand { get; set; }
         public DelegateCommand ToggleConsoleShowingCommand { get; set; }
+        // TODO: To be removed in Dynamo 2.0
+        [System.Obsolete("This delegate command is obsolete, set DynamoViewModel.ShowPreviewBubbles directly instead.")]
+        public DelegateCommand TogglePreviewBubblesShowingCommand { get; set; }
```

Comment:
hmm, so this delegate is back, but is it actually hooked up to the old method above anymore?